docs/pages/installation/kinD.md

```diff
@@ -20,9 +20,41 @@ chmod +x ./kind
 mv ./kind /some-dir-in-your-PATH/kind
 ```
 
-Create cluster using KinD
+Next we will demonstrate how to install Meshery with KinD on WSL2 
+
+### Create cluster using KinD
```

Comment:
Thank you for this, @tingkai-zhang. If the user is running KinD on Mac or Linux, they will need to execute steps inside of this "Create cluster using KinD" section, but only specific steps need to be executed in the event they are using WSL, right? Can we segregate the create cluster instructions based on OS?

Reply:
Yep, @leecalcote was right. There may multiple situations `MacOS/Linux` and `WSL`(I'm not sure, I saw many of the articles includes `WSL2`) .So.please kindly confirm `WSL2` or `WSL` is more fit the primary developer. And thanks for your contributions.

Reply:
Thanks for the review. I have updated the instructions.

Reply:
Thank you, @tingkai-zhang 👍